conf/portal.application.conf

```diff
@@ -214,7 +214,7 @@ hostProvider {
 # Alerts
 # ~~~~~
 alertRepository {
-  type = com.arpnetworking.metrics.portal.alerts.impl.NoAlertRepository
+  type = com.arpnetworking.metrics.portal.alerts.impl.DatabaseAlertRepository
```

Comment:
I think it should probably be off by default in including off for the ui functionality.

Reply:
Why are we changing this default?